src/main/java/com/trailfinder/dto/EventDTO.java

```diff
@@ -22,6 +21,38 @@
 	private ProfileDTO eventCreator;
 	private List<ProfileDTO> attendees;
 	
+	/**
```

Comment:
Overloaded constructor for initializing DTO with all fields during instantiation. Not sure which fields were vital, but you could overload more constructors if I included non-essential fields in this constructor.